controller.js

```diff
@@ -240,11 +239,11 @@ module.exports.createDish = (req, res) => {
   };
   Dish.create(dishData)
     .then((dish) => {
-      dish.setTags(req.body.dishTags).then(() => {
-        dish.setCategory(req.body.categoryId).then((data) => {
-          res.send(data);
-        });
-      });
+      return dish.setTags(req.body.dishTags)
+    }).then((dish) => {
+      return dish.setModifications(req.body.dishModifications)
+    }).then((dish) => {
+      res.send(dish);
```

Comment:
Maybe not in this PR but shall we consider using transactions for all like cases?

Reply:
Yeah - maybe we can include this in our overall menu service overhaul.  
I keep forgetting about transactions -_-